### ISSUE TITLE
refactor URI handler to only accept the version

### ISSUE THREAD

**Issue by matifali** [coder/jetbrains-coder#513]
With an end point like https://data.services.jetbrains.com/products?code=GO&type=release&majorVersion=2024.3&latest=true
Is it possible to get the download URL for the latest version?

Can we skip requiring the build number i.e. `ide_build_number` in the URI to launch JetBrains gateway?

**Comment by code-asher** [comment]
Makes sense, so if the build number is omitted we default to the latest.

**Comment by matifali** [comment]
> Makes sense, so if the build number is omitted we default to the latest.

Yes. We should default to latest.
